specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/backuprestore.json

```diff
@@ -18,15 +18,15 @@
           "FullBackup"
         ],
         "operationId": "FullBackup",
-        "description": "Creates full backup using the user provided SAS token to azure blob storage.",
+        "description": "Creates full backup using user provided SAS token to Azure Blob Storage container",
```

Comment:
Actually, "the" was correct - though "a" works better - but "user provided" used as an adjective should be highenated, i.e. "user-provided"; thus, "Creates a full backup using a user-provided SAS token to *an* Azure Blob Storage container." is best. Note the punctuation. This does generate documentation that doesn't typically add any (conditionally or otherwise).